src/scheduler/fifo.h

```diff
@@ -232,7 +232,7 @@ int scheduler_simulation_task(int pbs_sd, int debug);
 
 int update_svr_schedobj(int connector, int cmd, int alarm_time);
 
-int validate_sched_attrs(int connector);
+int set_validate_sched_attrs(int connector);
```

Comment:
Use extern keyword

Reply:
why?

Reply:
It is part of our [coding guidelines](https://pbspro.atlassian.net/wiki/spaces/DG/pages/1481670667/PBS+Pro+Coding+Standards). 

But it should be okay as none of the functions in this header file has extern in it.